metasfresh-webui-api/src/main/java/de/metas/ui/web/handlingunits/HUDocumentView.java

```diff
@@ -144,6 +154,16 @@ public int getM_HU_ID()
 		return (int)delegate.getFieldValueAsJson(I_WEBUI_HU_View.COLUMNNAME_M_HU_ID);
 	}
 
+	public I_M_HU getM_HU()
```

Comment:
my plan is to be able to wrap this with a proxy interface... like we do with our models... that's why I kept that name